docs/support.md

```diff
@@ -24,6 +26,8 @@ We encourage you to give back to the community by assisting other users via thes
 
 Lastly while Xen Orchestra is free and Open Source software, supporting and developing it takes a lot of effort. If you are considering using Xen Orchestra in production, please subscribe for one of our [professional support plans](https://xen-orchestra.com/#!/xo-pricing).
 
+> Note: support from the sources is harder, because Xen Orchestra can potentially run on any Linux distro (or even FreeBSD and Windows!). Always try to double check that you followed the guide on how to [install it from the sources](https://xen-orchestra.com/docs/from_the_sources.html) before going further.
```

Comment:
I would change "followed the guide" to "followed our guide" to highlight the fact we cannot guarantee the outcome of the many 3rd party XO sources deployment scripts out there.

Reply:
> I would change "followed the guide" to "followed our guide" to highlight the fact we cannot guarantee the outcome of the many 3rd party XO sources deployment scripts out there.

A script should have nothing like a guide. . . 

Run these commands in this order, is about all anything that I've seen states for "a guide". 

Mine included.